tests/api/ConnectedServer_Tests.cpp

```diff
@@ -164,8 +164,8 @@ TEST_CLASS(ConnectedServer_Tests)
         Assert::AreEqual(std::string(""), response.ErrorMessage);
         Assert::AreEqual(std::string(""), response.Username);
         Assert::AreEqual(std::string(""), response.ApiToken);
-        Assert::AreEqual(0U, response.Score);
-        Assert::AreEqual(0U, response.NumUnreadMessages);
+        Assert::AreEqual({0U}, response.Score);
+        Assert::AreEqual({0U}, response.NumUnreadMessages);
```

Comment:
`Score` and `NumUnreadMessages` are unsigned ints. Please don't wrap stuff in brackets unless it's really necessary. It makes the code harder to read as it implies a conversion that is not occurring, and it makes the diff harder to read as there's more stuff to sift through.